### PR TITLE
[15.0][MIG] hr_holidays_public

### PR DESCRIPTION
**Pull request by pegonzalezspesol**
*work done*
- do it installable
- tests passing
- fix pre-commit issues
- cherry-pick #36

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please include #36 and remember to put the proper PR title for being found.

File: hr_holidays_public/tests/test_holidays_calculation.py
Comment (pedrobaeza) [inline hr_holidays_public/tests/test_holidays_calculation.py]: You can remove these variables that doesn't serve for anything.

**Comment by pedrobaeza** [comment]
Please check pre-commit.

**Review by georginacicerchia — APPROVED** [review]
Fuctional review +1

File: hr_holidays_public/models/hr_leave.py
Comment (pedrobaeza) [inline hr_holidays_public/models/hr_leave.py:11]: This shouldn't be in this commit, but cherry picking the original commit

File: hr_holidays_public/models/hr_holidays_public.py
Comment (pedrobaeza) [inline hr_holidays_public/models/hr_holidays_public.py]: Why changing this?

Comment (pegonzalezspesol) [reply hr_holidays_public/models/hr_holidays_public.py]: > Why changing this?

to prevent next pylint-odoo error

image

if there is a better way of avoiding it let me know and I will implement it like that

File: hr_holidays_public/models/hr_leave.py
Comment (pegonzalezspesol) [reply hr_holidays_public/models/hr_leave.py:11]: Squashing I wrongly merged both commits cherry-picked and my work ... will look to see if I find a way of fixing it or I will create again the PR

File: hr_holidays_public/models/hr_holidays_public.py
Comment (pedrobaeza) [reply hr_holidays_public/models/hr_holidays_public.py]: You just need to remove `string="Name"`, not to change it. That's because the string is deduced from the field name.

Comment (pegonzalezspesol) [reply hr_holidays_public/models/hr_holidays_public.py]: > You just need to remove `string="Name"`, not to change it. That's because the string is deduced from the field name.

done!

**Comment by pedrobaeza** [comment]
Incorrect migration commit message.

**Review by nicomacr — APPROVED** [review]
LGTM, pending to change the commit message

**Comment by yankinmax** [comment]
@pedrobaeza can you pls take a look?

File: hr_holidays_public/models/hr_leave.py
Comment (pedrobaeza) [reply hr_holidays_public/models/hr_leave.py:11]: This is still pending

**Comment by AMhadhbi** [comment]
l10n_de_holidays is blocked by this pull request, will be good to merge it.
Thanks

**Comment by pedrobaeza** [comment]
There's a comment unattended: https://github.com/OCA/hr-holidays/pull/38#discussion_r782570860

You can supersede fixing such problem, but respecting attribution, and I'll merge it.

File: hr_holidays_public/models/hr_leave.py
Comment (pegonzalezspesol) [reply hr_holidays_public/models/hr_leave.py:11]: @pedrobaeza with my last PR

changes you commented are now chery-picked in commit

ca9063544ef691b9d14eab0bdb0102fc1e72ec77

my changes are in commit

c815f50c7277d0e42620ff34bafbcdd0eb980f26

**Comment by pedrobaeza** [comment]
Thanks

/ocabot migration hr_holidays_public
/ocabot merge nobump

**Pull request merged**
